packages/components/src/__future__/Workflow/_docs/Workflow.mdx

```diff
@@ -0,0 +1,167 @@
+import { ArgTypes, Canvas, Meta } from "@storybook/blocks"
+import { Installation, KaioNotification, ResourceLinks } from "../../../../../../storybook/components"
+import * as Workflow from "./Workflow.stories"
+import * as WorkflowFooter from "./WorkflowFooter.stories"
+import * as WorkflowHeader from "./WorkflowHeader.stories"
+
+<Meta title="Pages/Workflow" />
+
+# Workflow
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/Workflow"
+  className="!mb-8"
+/>
+
+<KaioNotification />
+
```

Comment:
Might want to pop an info in here to tell people that if they're running [insert older version] of KAIO where to find the docs. Just to make it complete 😄